invokeai/app/invocations/latent.py

```diff
@@ -231,7 +226,6 @@ def __init__(self):
             safety_checker=None,
             feature_extractor=None,
             requires_safety_checker=False,
-            precision="float16" if unet.dtype == torch.float16 else "float32",
```

Comment:
What problem was this causing, and is it right to remove?

Reply:
Pipeline doesn't make use of any such `precision` argument. We're passing in an already-loaded unet here, its data type has already been set.